### ISSUE TITLE
Add new default keys to pan, zoom and rotate

### ISSUE THREAD

**Issue by danboid** [mpv-player/mpv#5458]
I always add the following definitions to my input.conf. They're all very handy keyboard shortcuts to have so I'd like to see them added to the defaults.

```
Alt+RIGHT add video-rotate 90
Alt+LEFT add video-rotate -90
Alt+- add video-zoom -0.25
Alt+= add video-zoom 0.25
Alt+j add video-pan-x -0.05
Alt+l add video-pan-x 0.05
Alt+i add video-pan-y 0.05
Alt+k add video-pan-y -0.05
Alt+BS set video-zoom 0; set video-pan-x 0; set video-pan-y 0
```

**Comment by wm4** [comment]
Probably a good idea in theory...

**Comment by Dudemanguy** [comment]
We do actually have most of these in the default nowadays except `video-rotate` and the one that resets it all to zero.

**Comment by kasper93** [comment]
Fixed by #14350
